src/main/play-doc/operation/Install.md

```diff
@@ -255,19 +259,112 @@ On some Debian distributions you may need to add a dedicated Personal Package Ar
 
 ConductR provides a ConductR-HAProxy bundle that listens for bundle events from ConductR and updates the local HAProxy configuration file accordingly. We must specifically allow the bundle to use `sudo` to reload HAProxy.
 
-First, we have the user `conductr-agent` own the HAProxy config file.
+We have the user `conductr-agent` own the HAProxy config file.
+
+```bash
+[172.17.0.1]$ sudo chown conductr-agent:conductr-agent /etc/haproxy/haproxy.cfg
+```
+
+### Installing Docker based HAProxy
+
+We will use HAProxy version 1.5 official Docker image published to [Docker Hub](https://hub.docker.com/_/haproxy/).
+
+#### Prerequisite
+
+Docker needs to be installed on the host where HAProxy docker container will run.
+
+#### Preparing HAProxy config
+
+Setup basic HAProxy configuration for the Docker based HAProxy. Ensure the `conductr-agent` user which runs the ConductR-HAProxy has ownership to the HAProxy config file.
 
 ```bash
+[172.17.0.1]$ sudo mkdir /etc/haproxy
+[172.17.0.1]$ sudo touch /etc/haproxy/haproxy.cfg
 [172.17.0.1]$ sudo chown conductr-agent:conductr-agent /etc/haproxy/haproxy.cfg
 ```
 
+Populate the `/etc/haproxy/haproxy.cfg` file with the following configuration.
+
+```
+defaults
+    log global
+    mode    http
+    option  httplog
+    option  dontlognull
+    timeout connect 5000
+    timeout client  50000
+    timeout server  50000
+
+frontend monitor
+  bind :65535
+  mode http
+  monitor-uri /test
+```
+
+#### Running the HAProxy Docker container
+
+Run the HAProxy Docker container.
+
+```
+[172.17.0.1]$ docker run -d --name haproxy -p 10.0.7.118:9000:9000 -p 10.0.7.118:9999:9999 -p 10.0.7.118:65535:65535 -v /etc/haproxy/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg haproxy:1.5
+```
+
+The container has `haproxy` as its name.
+
+The proxy configuration located at `/etc/haproxy/haproxy.cfg` is mounted within the container on `/usr/local/etc/haproxy/haproxy.cfg`.
+
+The container exposes the port `9000`, `9999`, and `65535` to expose default port for ConductR HTTP-based endpoints, Visualizer bundle, and HAProxy test endpoint. Additional ports for your endpoints can be exposed through the `-p` option as required.
```

Comment:
I'd say: "The container exposes the ports `9000`, `9999` and `65535` to the host to make the defaults ports of the ConductR HTTP-based endpoints, the Visualizer bundle and the HAProxy test endpoint available."

Note that the port `65535` can be removed here in case you remove the `frontend monitor` configuration in the `haproxy.cfg` as well, as Chris suggested in https://github.com/typesafehub/conductr-doc/pull/251/files#r75637541.